functions/two_sum.py

```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
```

Comment:
I think you did a great job and don't let anyone else tell you otherwise. *cough cough* Fena *cough*